### PR TITLE
Ja feature code cleanup and proptypes

### PR DESCRIPTION
**Pull request by jeremiahaguirre**
- added splash screen

### CODE REVIEW

File: client/ios/Podfile [hunk 1]
```diff
@@ -40,6 +40,9 @@ target 'bikelockerSpring2019' do
 
     pod 'rn-fetch-blob', :path => '../node_modules/rn-fetch-blob'
 
+
+    pod 'react-native-splash-screen', :path => '../node_modules/react-native-splash-screen'
+
```
Comment (siddharthparmar7) [inline client/ios/Podfile:45]: did you install splash screen package using pods?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work!

File: client/ios/Podfile (hunk 1 above)
Comment (jeremiahaguirre) [reply client/ios/Podfile:45]: No I didn't

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

File: client/js/screens/AddLocker/styles.js [hunk 2]
```diff
@@ -25,7 +30,16 @@ const styles = StyleSheet.create({
...
+<<<<<<< HEAD
+    fontFamily: theme.bodyFont,
+    color: "#fff",
+    flex: 1,
+    justifyContent: "center",
+    flexWrap: "wrap",
+    textAlign: "center"
+=======
     color: "#fff"
+>>>>>>> develop
```
Comment (siddharthparmar7) [inline client/js/screens/AddLocker/styles.js]: resolve this merge conflicts.

File: client/js/screens/AddLocker/styles.js [hunk 3]
```diff
@@ -39,25 +53,26 @@ const styles = StyleSheet.create({
...
     fontSize: 20,
     fontWeight: "bold",
     padding: 10,
     width: "100%"
   },
   ratingText: {
+<<<<<<< HEAD
+    fontFamily: theme.bodyFont,
+=======
+>>>>>>> develop
```
Comment (siddharthparmar7) [inline client/js/screens/AddLocker/styles.js]: same as above

File: client/js/screens/AddLocker/styles.js [hunk 4]
```diff
@@ -67,10 +82,15 @@ const styles = StyleSheet.create({
     justifyContent: "space-between"
   },
   secureText: {
+<<<<<<< HEAD
+    fontFamily: theme.bodyFont,
+=======
+>>>>>>> develop
```
Comment (siddharthparmar7) [inline client/js/screens/AddLocker/styles.js]: same as above.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**